src/reducers/errorReducer.test.js

```diff
@@ -0,0 +1,38 @@
+import * as ActionTypes from '../actions/actionTypes';
+import reducer from './errorReducer';
+import initialState from './initialState';
+
+ describe('Reducers :: Error', () => {
+  const getInitialState = initialState.error;
+  it('should get initial state by default', () => {
+    const action = { type: 'unknown' };
+    const expected= getInitialState;
+    expect(reducer(undefined, action)).toEqual(expected);
+  });
+  it('should handle Error Actions', () => {
+    const action = { type: ActionTypes.ERROR_ACTION, error: {status: true}};
+    const expected = Object.assign({}, getInitialState.status, action.error.status);
+    expect(reducer(getInitialState, action)).toEqual(expected);
+  });
+  it('should handle FETCH_INCIDENT', () => {
+    const action = { type: ActionTypes.FETCH_INCIDENT, error: {status: true}};
```

Comment:
The style here should be consistent with line 24 and beyond.

Reply:
done